dotcms-integration/src/test/java/com/dotcms/experiments/business/web/ExperimentWebAPIImplIntegrationTest.java

```diff
@@ -1061,4 +1092,108 @@ public void excludeExperiments() throws DotDataException, DotSecurityException {
             }
         }
     }
+
+    private HTMLPageAsset getHtmlPageAsset(final Experiment experiment) {
+
+        try {
+            final Contentlet contentlet = APILocator.getContentletAPI()
+                    .findContentletByIdentifierAnyLanguage(experiment.pageId(), false);
+            final HTMLPageAsset htmlPageAsset = APILocator.getHTMLPageAssetAPI()
+                    .fromContentlet(contentlet);
+            return htmlPageAsset;
+        } catch (DotDataException e) {
+            throw new DotRuntimeException(e);
+        }
+    }
+
+    /**
+     * Method to test: {@link ExperimentWebAPIImpl#isUserIncluded(HttpServletRequest, HttpServletResponse, List)}
+     * when: You have two sites let call them A y B, both with an index page, but just //A/index have an Experiment Running
+     * should: when call the method with the A as currentSite should take include the Experiment but when it is called
+     * with B as current Site then the Experiment should not be take account
+     */
+    @Test
+    public void runningExperimentOnOneSite() throws DotDataException, DotSecurityException {
+        final Host siteA = new SiteDataGen().nextPersisted();
+        final Template template = new TemplateDataGen().nextPersisted();
+        final HTMLPageAsset pageA = new HTMLPageDataGen(siteA, template).nextPersisted();
+
+
+        final Host siteB = new SiteDataGen().nextPersisted();
+        final HTMLPageAsset pageB = new HTMLPageDataGen(siteB, template).nextPersisted();
+
+        final Experiment experiment = new ExperimentDataGen().page(pageA).nextPersisted();
+
+        APILocator.getExperimentsAPI().start(experiment.id().get(), APILocator.systemUser());
+
+        final HttpServletRequest request = mock(HttpServletRequest.class);
+        when(request.getParameter("host_id")).thenReturn(siteB.getIdentifier());
+
+        final HttpServletResponse response = mock(HttpServletResponse.class);
+
+        final SelectedExperiments userIncluded = WebAPILocator.getExperimentWebAPI().isUserIncluded(request, response,
+                Collections.EMPTY_LIST);
+
+        assertTrue(userIncluded.getIncludedExperimentIds().isEmpty());
+        assertTrue(userIncluded.getExperiments().isEmpty());
+        assertTrue(userIncluded.getExcludedExperimentIds().isEmpty());
```

Comment:
Looks like you're missing checking that siteA has includedExperiments

Reply:
done https://github.com/dotCMS/core/pull/27989/commits/b1d927cd36b8975d11b2c4b0e92e3ee543e41133#diff-d2d06a01769a4269ebb8d5654c69294b6eded1d961a6dd0677ac1840459ee874R1131-R1149